build/commands/lib/config.js

```diff
@@ -756,6 +756,11 @@ Config.prototype.buildArgs = function () {
       args.android_static_analysis = 'on'
     }
 
+    // Align DCHECKs with Java asserts
+    if (args.dcheck_always_on === false) {
```

Comment:
isn't this already aligned `enable_java_asserts = dcheck_always_on || !is_official_build`?

Reply:
oic, nvm

Reply:
but curious why we want to do something different here than upstream?

Reply:
I suspect it crashes emulator on our CI during Android browser test at so bad way, that there are even no logcat files  `TEST_RESULTS_***` folder.